src/app/grid/grid-paste/grid-paste-sample.component.ts

```diff
@@ -16,7 +16,7 @@ import { PasteHandler} from "./paste-handler.directive";
 
 import { EXCEL_DATA, LOCAL_DATA } from "./data";
 
-import { take } from "rxjs/operators";
+import { first, take } from "rxjs/operators";
 
 @Component({
     encapsulation: ViewEncapsulation.None,
```

Comment:
Since I can comment on this now - no idea what the sample is doing without encapsulation, but it shouldn't. This should be removed